main.go

```diff
@@ -265,7 +265,7 @@ func main() {
 		},
 		&cli.BoolFlag{
 			Name:  "no-storage-cron",
```

Comment:
I would like to see this flag's name changed too. `storage-cron` doesn't tell me much